functions/zoom-uploader.py

```diff
@@ -210,7 +210,12 @@ def workflow_definition_id(self):
 
     @property
     def s3_filenames(self):
-        return self.data["s3_filenames"]
+        if not hasattr(self, "_s3_filenames"):
+            self._s3_filenames = {}
+            for view, file in self.data["s3_files"].items():
+                self._s3_filenames[view] = [data["filename"] for data in file["segments"]]
+
+        return self._s3_filenames
 
```

Comment:
It just seemed easiest, rather than mess with the uploader code, to give it the data structure it expects.